### DISCUSSION TITLE
LineageOS-20.0 sync failing

### DISCUSSION THREAD

**Discussion by SwapnilVicky** [ROM-builders/temporary#18678] [General]
Build 1: https://cirrus-ci.com/task/5835733009694720
Build 2: https://cirrus-ci.com/build/5021827244752896

**Comment by Apon77** [comment]
bliss is for your tree reason probably

**Reply by SwapnilVicky** [reply]
Happening because i have a base of Bliss trees ?

**Reply by SwapnilVicky** [reply]
again got the sync error https://cirrus-ci.com/task/5471310336229376
